### PR TITLE
feat: Verify Identity From Metric

### PR DESCRIPTION
**Pull request by Bendomey**

### CODE REVIEW

File: services/main/DTOs/IdentityVerificationInput.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    public required string Msg { get; set; }
+
+    [JsonPropertyName("error")]
+    [BsonElement("error")]
+    public string? Error { get; set; }
+
+    [JsonPropertyName("nationalId")]
+    [BsonElement("national_id")]
+    public required string NationalId { get; set; }
+}
```
Comment (Bendomey) [inline services/main/DTOs/IdentityVerificationInput.cs:84]: Metric Structure I could get from logs.... we can only test to confirm if it works as it supposed to

File: services/main/DTOs/IdentityVerificationInput.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    [BsonElement("verification_result")]
+    public VerificationResult? VerificationResult { get; set; }
+}
+
+public class VerificationResult
+{
+    [JsonPropertyName("success")]
+    [BsonElement("success")]
+    public required bool Success { get; set; }
+
```
Comment (arkhurst) [inline services/main/DTOs/IdentityVerificationInput.cs:64]: not sure if you've seen this linting error

**Review by arkhurst — COMMENTED** [review]
just saw a couple of linting errors, aside that LGTM

**Review by arkhurst — COMMENTED** [review]
just saw a couple of linting errors, aside that LGTM

File: services/main/DTOs/IdentityVerificationInput.cs (hunk 2 above)
Comment (Bendomey) [reply services/main/DTOs/IdentityVerificationInput.cs:64]: Yeah, just pushed a commit

**Review by arkhurst — APPROVED** [review]

**Pull request merged**